src/npctalk.cpp

```diff
@@ -2980,6 +2989,16 @@ void talk_function::stop_guard( npc &p )
     p.guard_pos = npc::no_goal_point;
 }
 
+void talk_function::become_overseer( npc &p )
+{
+    add_msg( _( "%s has become a camp manager." ), p.name.c_str() );
+    p.name = p.name + ", Camp Manager";
```

Comment:
Sholdn't the title be translated as well? (Isn't "Camp Manager" supposed to be a title?)

Reply:
Yes... but for the moment factions are still determining roles based on title instead of a separate variable.  100% should be fixed so that it is translation compatible and renaming NPCs is cool but that requires going through and ensuring compatibility for all older outstanding missions.